kore/src/Kore/ModelChecker/Step.hs

```diff
@@ -172,7 +172,7 @@ transitionRule
 
     applyUnroll ModalPattern { modalOp, term } wrapper config
       = case modalOp of
-            "ag" -> do
+            "allPathGlobally" -> do
```

Comment:
Let's define a constant for this somewhere.

Reply:
done